### ISSUE TITLE
Navigability issues in DG

### ISSUE THREAD

**Issue by tingyic** [tingyic/pe#14]
There is the "Return to ToC" button in the UG, but not in the DG. Not very user friendly, especially when there are 72 pages in the DG.

**Comment by nus-pe-script** [comment]
# Team's Response
We do find merit in your suggestion. However, the reason why we included `back to TOC` in our UG is because we felt that users reading the UG are likely to want to jump around to different sections (especially the command manual) a lot as they are executing sequences of commands for the first time. `back to TOC` makes it easier for them to do so.

When it comes to the DG, we expect developers to be more focused on just one section for a long time and not jump around as much as a user would when reading  the UG. Hence, it was not as high of a priority despite the DG being longer. We do think the suggestion is good but "rectifying it is less important than the work that has been done already". Currently, it is possible to obtain a similar functionality by using the scroll bar. Nevertheless, we would like to add this in the future, hence, we think it is not in scope.
# Items for the Tester to Verify
## :question: Issue response

Team chose [`response.NotInScope`]

- [x] I disagree

**Reason for disagreement:** Thank you for your feedback. As much as I agree with your point that says that developers will tend to focus on one section for a prolonged period, and will not jump around much, but I still think that the Return to ToC will still be an important feature that improves readability, since developers may focus on multiple sections for a prolonged period. For example, one may be interested to find out how the import feature work, followed by the navigation component, and then finally to the appendix for manual testing.

Of course, this is nitpicking, and I apologize if I offended you. However, despite this, the example above could very likely occur, and my point of having 72 pages in the DG is still valid.

In conclusion, I believe you should accept this, and it will not be a difficult implement it, since you have already done something similar in your UG.


<catcher-end-of-segment><hr>
